lib/jsdom/MultihandleWeakMap.js

```diff
@@ -0,0 +1,49 @@
+﻿"use strict";
+
+var instances = new WeakMap();
+
+function MultihandleWeakMap() {
```

Comment:
As discussed over IM we should probably use symbols instead of weak maps for now.